.gitlab-ci.yml

```diff
@@ -337,12 +336,15 @@ unleash-check:
   <<:                              *test-refs-no-trigger
   script:
     - cargo install cargo-unleash ${CARGO_UNLEASH_INSTALL_PARAMS}
```

Comment:
Is it mature enough to be installed into CI image?

Reply:
Probably - until it's not v1 I'd prefer not to; the time savings is miniscule I'd say comparing to the actual checking done by the tool later on

Reply:
not on every commit tho